build-system/tasks/dep-check.js

```diff
@@ -204,9 +203,12 @@ function getGraph(entryModule) {
  */
 function getEntryModule(extensionFolder) {
   var extension = path.basename(extensionFolder);
-  var versions = extensionsVersions[extension] || ['0.1'];
-  return versions.map(v =>
-      `${extensionFolder}/${v}/${extension}.js`);
+  return fs.readdirAsync(extensionFolder).then(dirItems => {
```

Comment:
Make sense. But for `dep_check` I would get error like this when running locally.

> Cannot find module '../../../build/amp-sticky-ad-0.1.css' from '/Users/zhouyx/amphtml/extensions/amp-sticky-ad/0.1'

It is because we no longer compile css for 0.1 when running `gulp css`. I am surprised this passed travis. Could you double check locally? Run `gulp dep-check` after `gulp clean`? 
Other than that I am good with adding dep check to every version.